installation/cmd/run.sh

```diff
@@ -126,7 +126,7 @@ fi
 
 echo "Label Minikube node for benchmark execution..."
 NODE=$(kubectl get nodes | tail -n 1 | cut -d ' ' -f 1)
-kubectl label node "$NODE" benchmark=true
+kubectl label node "$NODE" benchmark=true || true
```

Comment:
Minor: Why is this necessary?

Reply:
If you run installation with --skip-minikube-start the node is already labelled.